lib/playground.dart

```diff
@@ -693,6 +705,22 @@ class Playground implements GistContainer, GistController {
     });
   }
 
+  Future _format() {
+    SourceRequest input = new SourceRequest()..source = _context.dartSource;
+
+    Future request = dartServices.format(input).timeout(serviceCallTimeout);
+    return request.then((FormatResponse result) {
+      busyLight.reset();
```

Comment:
I can't remember if we have a good toast UI in dartpad. If so, perhaps a toast that auto-dismisses after a brief period (`'Format successful.'`; `'No formatting changes.'`).